### ISSUE TITLE
Detected call of `lr_scheduler.step()` before `optimizer.step()`

### ISSUE THREAD

**Issue by Shadow-Alex** [ultralytics/ultralytics#3326]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

When training on multiple gpu, my script looks like this:
```python
from ultralytics import YOLO

model = YOLO('yolov8n.yaml')

results = model.train(data='cdla.yaml', epochs=300, device=[4,5,6,7], batch=512)
```

I encountered a warning : lib/python3.8/site-packages/torch/optim/lr_scheduler.py:129: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate.

It seems 3 out of 4 GPUs are running, the first gpu, which is gpu4 in my case, is kind of idling.
```
+-------------------------------+----------------------+----------------------+
|   3  TITAN RTX           Off  | 00000000:3E:00.0 Off |                  N/A |
| 40%   46C    P8    18W / 280W |     11MiB / 24220MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   4  TITAN RTX           Off  | 00000000:88:00.0 Off |                  N/A |
| 48%   66C    P2    91W / 280W |  20906MiB / 24220MiB |      2%      Default |
+-------------------------------+----------------------+----------------------+
|   5  TITAN RTX           Off  | 00000000:89:00.0 Off |                  N/A |
| 55%   74C    P2   124W / 280W |   1790MiB / 24220MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
|   6  TITAN RTX           Off  | 00000000:B1:00.0 Off |                  N/A |
| 53%   73C    P2   114W / 280W |   1824MiB / 24220MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
|   7  TITAN RTX           Off  | 00000000:B2:00.0 Off |                  N/A |
| 53%   72C    P2   123W / 280W |   1792MiB / 24220MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
```
Should I worry about the warning?

### Environment

- YOLO: v8.0.121
- Python: 3.8.13
- torch: 1.8.0
- OS: ubuntu

### Minimal Reproducible Example

```python
from ultralytics import YOLO

model = YOLO('yolov8n.yaml')

results = model.train(data='cdla.yaml', epochs=300, device=[4,5,6,7], batch=512)
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@Shadow-Alex you can safely ignore this warning, which occurs on some versions of torch. It's absent from all of my macOS trainings and I never see it in Colab, so I don't think there is any issue.
